src/app/components/typeFilter/useModel.ts

```
@@ -12,124 +12,124 @@ import { useResetAtom } from "jotai/utils";
+
+	useEffect(() => {
+		// selectedTypesまたはselectedTypePokemonListが変更された場合のみ、フィルタリストを設定する
+		setSelectedTypePokemonFilterListAtom(selectedTypePokemonList);
+	}, [selectedTypePokemonList]);
+
+	useEffect(() => {
+		if (selectedTypes.length === 1) {
+			// selectedTypes が 1 つの場合、全てのポケモンリストを適用
+			setSelectedTypePokemonFilterListAtom(selectedTypePokemonList);
+		} else if (selectedTypes.length >= 2) {
+			// selectedTypes が 2 つ以上の場合、フィルタリングされたリストを適用
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: setSelectedTypePokemonFilterListAtom

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)